### PR TITLE
add README.jp.md

### PR DESCRIPTION
**Pull request by yamatoya**
https://github.com/spectresystems/spectre.console/pull/107

I add a localized README for Japanese.

### CODE REVIEW

File: README.jp.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+`Spectre.Console` APIはステートフルで、スレッドセーフではありません。
+異なるスレッドからコンソールに書く必要がある場合、通常の`System.Console` APIを使用するときと同様、適切な注意を払ってください。
+
+現在の端末がANSIエスケープシーケンスに対応していない場合、
+`Spectre.Console`は、`System.Console` APIの利用に切り替わります。
+
+_メモ: このライブラリは現在開発中で、APIは1.0のリリースまでの間に変更されたり、
+削除されたりする可能性があります。_
+
+### static APIの利用
```
Comment (patriksvensson) [inline README.jp.md:40]:
```suggestion
### Static APIの利用
```

File: README.jp.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+綺麗なコンソールアプリケーションを簡単に作成するための.NET Standard 2.0ライブラリです。
+Python用の素晴らしい[Rich ライブラリ](https://github.com/willmcgugan/rich)に強く影響を受けています。
+
+## 目次
+
+1. [特徴](#features)
+2. [例](#example)
+3. [使用方法](#usage)  
+   3.1. [static APIの利用](#using-the-static-api)
```
Comment (patriksvensson) [inline README.jp.md:13]:
```suggestion
   3.1. [Static APIの利用](#using-the-static-api)  
```

File: README.jp.md [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+綺麗なコンソールアプリケーションを簡単に作成するための.NET Standard 2.0ライブラリです。
+Python用の素晴らしい[Rich ライブラリ](https://github.com/willmcgugan/rich)に強く影響を受けています。
+
+## 目次
+
+1. [特徴](#features)
+2. [例](#example)
+3. [使用方法](#usage)  
+   3.1. [static APIの利用](#using-the-static-api)  
+   3.2. [consoleの作成](#creating-a-console)
```
Comment (patriksvensson) [inline README.jp.md:14]: I can't find this header below, did you forget to change this one?

**Review by patriksvensson — CHANGES_REQUESTED** [review]
Awesome work!

Don't we need to update the anchor (like `2. [例](#example)`) for the links to work as well?

File: README.jp.md (hunk 3 above)
Comment (patriksvensson) [inline README.jp.md:14]:
```suggestion
   3.2. [Consoleの作成](#creating-a-console)
```

**Review by patriksvensson — CHANGES_REQUESTED** [review]

File: README.jp.md (hunk 3 above)
Comment (yamatoya) [reply README.jp.md]: I fix.

**Pull request merged**